pywren/executor.py

```diff
@@ -176,27 +166,25 @@ def map(self, func, iterdata, extra_env = None, extra_meta = None,
         host_job_meta = {}
 
         pool = ThreadPool(invoke_pool_threads)
-        callset_id = s3util.create_callset_id()
+        callset_id = wrenutil.create_callset_id()
+        data = list(iterdata)
```

Comment:
`s3util.create_callset_id` is just a wrapper around `wrenutil.create_callset_id`